package/src/lib/SpotlightTour.context.ts

```diff
@@ -114,6 +89,11 @@ export interface TourStep {
    * @default undefined
    */
   before?: () => void | Promise<void>;
+  /**
+   * Specifies {@link FloatingProps} in order to configure Floating UI
+   * in a specific tour step layout.
+   */
```

Comment:
Maybe we should add a `@default` to the docs here so people know what they have if they omit this prop 🙂